core/genesis.go

```diff
@@ -568,19 +568,17 @@ func DefaultSepoliaGenesisBlock() *Genesis {
 // DeveloperGenesisBlock returns the 'geth --dev' genesis block.
 func DeveloperGenesisBlock(period uint64, gasLimit uint64, faucet common.Address) *Genesis {
 	// Override the default period to the user requested one
-	config := *params.AllCliqueProtocolChanges
-	config.Clique = &params.CliqueConfig{
+	config := *params.AllDevChainProtocolChanges
```

Comment:
not sure why you remove `AllCliqueProtocolChanges` here?

Reply:
Because developer mode no longer uses Clique for its consensus engine.